apps/devhub/utils.py

```diff
@@ -1,8 +1,21 @@
 from django.conf import settings
+from django.db.models import Q
+from django.forms import ValidationError
 
+from celery import chain, group
 from tower import ugettext as _
 
+import amo
+from addons.models import Addon
 from amo.utils import escape_all
+from files.models import File, FileUpload
+from files.utils import parse_addon
+from validator.version import Version
+from . import tasks
+
+
+# FIXME: Move to `validator.constants`.
+SEVERITIES = {'trivial', 'low', 'medium', 'high'}
```

Comment:
I tried that, but that property isn't available on the `ErrorBundler` class, only instances. I want to move it to `validator.constants`, and use it from there both inside the validator and here, but I don't want this PR to rely on validator changes, so I'm pushing that change back.